docs/contributing/managing-pull-requests.md

```diff
@@ -140,6 +142,18 @@ These are bad PR titles because they are generic, don't communicate the change p
 - Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
 - Don't suggest and expect changes out of scope which are best addressed in a separate PR
 
+## Pushing changes to a remote fork
+
+Sometimes the easiest way to unblock a stalled PR is to sort out merge conflicts or apply remaining suggestions. When the GitHub UI won't cut it, you can (often) apply changes directly to someone's remote fork with Git:
+
+- Add their Gatsby fork as a remote:<br />`git add <forkname> git@github.com:<username>/gatsby.git`
+- Fetch the branches:<br />`git fetch <forkname>`
+- Check out their branch locally:<br />`git checkout -b <branch-name> <forkname>/<branch-name>`
```

Comment:
```suggestion
- Check out their branch locally:<br />`git checkout <branch-name> <forkname>/<branch-name>`
```
Again, could be wrong, but I think -b creates the branch. If you want to checkout an existing branch you don't want it.

Reply:
I believe it does need the `-b` to check out the branch from a remote fork, at least that's how I've gotten it to work when typing it out by hand (and I've done it a lot)

Reply:
As long as it’s intentional and works that’s good with me!